ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/ChainSync/Client/Jumping.hs

```diff
@@ -75,6 +75,13 @@
 -- when the client should pause, download headers, or ask about agreement with
 -- a given point (jumping). See the 'Jumping' type for more details.
 --
+-- Interactions with the BlockFetch logic
+-- --------------------------------------
+--
+-- When syncing, the BlockFetch logic will fetch blocks from the dynamo. If the
+-- dynamo is responding too slowly, the BlockFetch logic can ask to change the
+-- dynamo with a call to 'rotateDynamo'.
+--
```

Comment:
Should we mention here that this is only allowed to happen on honest peers (with “honest” defined as the peer from the HAA)?

Reply:
This text needs to be revised, as it may not always be the case that blocks are downloaded from the dynamo.

Reply:
Indeed. Frankly, I also don't understand my initial message. I suppose it is missing a negation, `rotateDynamo` is supposed to _not_ be used on the honest peer.